sklearn/neural_network/multilayer_perceptron.py

```diff
@@ -619,7 +619,8 @@ def fit(self, X, y):
 
     @property
     def partial_fit(self):
-        """Fit the model to data matrix X and target y.
+        """Fit the model to data matrix X and target y for a single
```

Comment:
How about "update the model with a single iteration over the given data"
This would also satisfy pep257's request that we keep this summary to a single line

Reply:
I like it, done.